molgenis-data-annotators/src/main/java/org/molgenis/data/support/AnnotatorDependencyOrderResolver.java

```diff
@@ -68,19 +75,20 @@ private void resolveAnnotatorDependencies(RepositoryAnnotator selectedAnnotator,
 		}
 		if (annotatorQueue.size() == 0)
 		{
+			// FIXME: what to do for ref entity annotator.
```

Comment:
Potentially very complex, not needed now. Workaround works fine for current situation but it will be a nice enhancement for the future.